key-manager-test/src/test/scala/org/bitcoins/keymanager/bip39/BIP39KeyManagerApiTest.scala

```diff
@@ -46,21 +46,23 @@ class BIP39KeyManagerApiTest extends KeyManagerApiUnitTest {
 
   it must "initialize the key manager" in {
     val entropy = MnemonicCode.getEntropy256Bits
-    val keyManager = withInitializedKeyManager(entropy = entropy)
+    val aesPasswordOpt = KeyManagerTestUtil.aesPasswordOpt
```

Comment:
nit: So I think I would _prefer_ explicit test cases for the combinations of `aesPassword` being defined and `aesPassword == None`. 

1. It makes it easier to debug if we do introduce a regression
2. We now have determinism in these test cases, since `KeyManagerTestUtil.aesPasswordOpt` can be `Some` or `None`, it will only fail somes times if we introduce a regression.
3. This is a sensitive part of the code base, so i think it's best to be explicit as possible with the tests.

Reply:
I think it is okay as is in this case because the aes password is only used when reading and writing from the disk. All the possible cases are handled in the `WalletStorageTest` and then the other key manager tests shouldn't care if it reading from an encrypted or unencrypted seed